deepchecks/vision/utils/detection_formatters.py

```diff
@@ -203,3 +207,176 @@ def validate_prediction(self, batch_predictions, n_classes: int, eps: float = 1e
         if batch_predictions[0].shape[1] != 6:
             raise DeepchecksValueError('Check requires detection predictions to be a list of 2D tensors, when '
                                        'each row has 6 columns: [x, y, width, height, class_probability, class_id]')
+
+
+def verify_bbox_format_notation(notation: str) -> List[str]:
```

Comment:
We're missing one important format option here (sorry, my bad for forgetting to detail it in the issue) - we need an "n" option, which means that all coordinates are normalized rel to image size, rather than the default "absolute" coordinates which are in pixels.

Reply:
ok, I will add

Reply:
added